src/Contract/CircuitBreakerInterface.php

```diff
@@ -18,11 +18,12 @@ public function getState();
      * The function that execute the service.
      *
      * @param string $service the service to call
-     * @param callable $fallback if the service is unavailable, rely on the fallback
+     * @param array $parameters the parameters for the request
+     * @param callable|null $fallback if the service is unavailable, rely on the fallback
      *
      * @return string
      */
-    public function call($service, callable $fallback);
+    public function call($service, array $parameters = [], callable $fallback = null);
```

Comment:
It's a BC 🤔

Reply:
Yes it is, we will increase the major version. This was allowed by @eternoendless as the lib is not much used for now, this the moment to allow ourselves some BC breaks.